eng/pipelines/docindex.yml

```diff
@@ -49,14 +50,29 @@ jobs:
       - template: /eng/common/pipelines/templates/steps/set-daily-docs-branch-name.yml
         parameters:
           DailyBranchVariableName: DailyDocsBranchName
+      # Create a branch to track docs fork repo onboarding files/folders
       - pwsh: |
           $ErrorActionPreference = "Continue"
-          git checkout "origin/$(DailyDocsBranchName)" 2>&1 | Out-Null
-          $LASTEXITCODE = 0 # This ignores any error from git checkout
-          git status
-        displayName: Checkout daily branch if it exists
+          # git remote on fork repo
```

Comment:
Yes this is more along the lines of what I was expecting to see. 

As a side question what is the process going to be to move the branch from the fork to the main repo? Is that going to happen manually or as part of some other process?

Reply:
No, we do not have the process of moving daily branch to main in Azure repo. They are independent. Daily branch is for preview the doc purpose and they are using alpha version which is not for main branch

Reply:
The git-push-branch will push fork daily branch to Azure daily branch. That's the purpose of docindex.

Azure daily is independent of Azure main.

Reply:
OK I understand now.